qutebrowser/config/configdata.yml

```diff
@@ -469,6 +469,19 @@ content.javascript.prompt:
   type: Bool
   desc: Show javascript prompts.
 
+content.javascript.userjs_directory:
```

Comment:
Hmm, I'm not sure whether it makes sense to make this configurable really... Do you have an use-case for that? If not, I'd rather just use the datadir without having a setting for it.